tests/uricontent/uricontent-03/README.md

```diff
@@ -0,0 +1 @@
+Tests the modifiers for uricontent and content match
```

Comment:
This and the following two tests have the same description, do they do nothing differently?

Reply:
No. I think I should have converted them to a single test.